### ISSUE TITLE
[Affected functionality] Sonar Scan Fails During Parallel Pull Request Builds in Jenkins

### ISSUE THREAD

**Issue by YaroslavRosnovskyi** [ita-social-projects/StreetCode#1922]
**Environment:** Windows 11 Home
**Reproducible:** always
**Build found:** last commit from https://github.com/ita-social-projects/StreetCode_Client/pull/1489/commits/62f837c592707ebbd16f74bbd7d069f7bd270db0

**Preconditions**
Prepare two separate pull requests for the project repository.

**Steps to reproduce**
1. In Jenkins trigger the build (for example Pull Request 1489).
2. Immediately trigger the build for another pull request (for example Pull Request 1488) so that both builds run in parallel.

**Actual result**
The second build fails when trying to execute the Sonar scan step while the first build is still in the Sonar scan step. The logs for the second build contain the following errors:

1. Embedded Node.js Failed to Deploy:
java.nio.file.FileSystemException: /home/admin_1/.sonar/js/node-runtime/node: Text file busy.
2. Unsupported Node.js Version:
Error: Unsupported Node.JS version detected 10.19. Please upgrade to the latest Node.JS LTS version.
3. SonarScanner Analysis Error:
Error while running Node.js. A supported version of Node.js is required for running the analysis of JS/TS files.

[Log link](http://138.91.120.146:8080/job/New_Frontend_PIpline/view/change-requests/job/PR-1488/11/execution/node/68/log/) 

**Expected result**
Both pull request builds should complete successfully, and SonarScanner should analyze the code for both builds without any errors, even when running in parallel.

**Postconditions**
Stop any running builds if necessary.
